kymatio/scattering2d/backend/torch_backend.py

```diff
@@ -223,12 +223,12 @@ def fft(x, direction='C2C', inverse=False):
         raise RuntimeError('Tensors must be contiguous.')
 
     if direction == 'C2R':
-        output = torch.irfft(x, 2, normalized=False, onesided=False) * x.shape[-2] * x.shape[-3]
+        output = torch.irfft(x, 2, normalized=True, onesided=False)
```

Comment:
The unitary FFT, really? This is quite non-standard. Why not stick to the regular ones?

Reply:
well, the dotwise multiplication is anyway slow - i'm also ok to remove the "normalized" fft

Reply:
Should be faster then with `normalized=False` since that's only one elementwise multiplication (in the IFFT).